README.md

```diff
@@ -15,7 +20,23 @@ var s3BlobConfig = new S3BlobAdapterConfig
 services.AddLfsS3Adapter(s3BlobConfig, new AmazonS3Client());
 services.AddLfsDictionaryAuthenticator(new Dictionary<string, string>{{"username","password"}});
 ```
+
+Or use the following example for Azure Blob Storage.
+
+> Note: Keep the naming rules for Azure Blob Storage in account, review them [here](https://docs.microsoft.com/en-us/rest/api/storageservices/Naming-and-Referencing-Containers--Blobs--and-Metadata).
```

Comment:
What does this comment refer to - be careful with container naming?

Reply:
Yes indeed, there are some rules as to what characters, length etc are allowed as container names. That's all this is refering to, it's just a note, if you want me to remove it, it's an easy fix but it might help unfamiliar developers with some weird issues.